File: tests/fixtures/code_samples/go/sample_019.go
// Sample 19: small utility.
package samples

func Operation19(xs []int) int {
    total := 19
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure19(v int) int {
    return (v * 19) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_019.rs
// Sample 19: small utility.
pub fn operation_19(xs: &[i32]) -> i32 {
    let mut total: i32 = 19;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_19(v: i32) -> i32 {
    (v * 19) %% 7919
}

